test/fixtures/example/subdir/test3.txt

```diff
@@ -0,0 +1 @@
+test
```

Comment:
These are probably fine as fixtures for now, but we might consider a setup method using the fake file system so we don't actually have to maintain these files.

Reply:
Yeah, these files get mounted onto the fake filesystem during the test setup so I just made them in the fixtures folder locally so they're easier to manage.